src/launcher/Launcher.cc

```diff
@@ -673,3 +673,41 @@ void Launcher::moveDownMods(Fl_Widget* widget, void* userdata) {
 	window->enabledModsBrowser->redraw();
 	window->update(true, widget);
 }
+
+void Launcher::selectGameVersion(Fl_Widget* widget, void* userdata)
+{
+	Launcher* window = static_cast<Launcher*>(userdata);
+	int currentResourceVersionIndex = window->gameVersionInput->value();
+	GameVersion currentResourceVersion = predefinedVersions.at(currentResourceVersionIndex);
+	if (currentResourceVersion == GameVersion::SIMPLIFIED_CHINESE && !window->checkSimplifiedChineseMod())
+	{
+		//force enalbe Simplified Chinese Mod
+		for (auto i = window->availableModsBrowser->size(); i > 0; i--)
+		{
+			char* modId = static_cast<char*>(window->availableModsBrowser->data(i));
+			if (strcmp(modId, SIMPLIFIED_CHINESE_MOD_NAME) == 0)
+			{
+				window->enabledModsBrowser->insert(1, window->availableModsBrowser->text(i));
+				window->enabledModsBrowser->data(1, window->availableModsBrowser->data(i));
+				window->enabledModsBrowser->select(1, 1);
+				window->availableModsBrowser->hide(i);
+				break;
+			}
+		}
+	}
```

Comment:
This should also do the opposite, disable the mod when a non-chinese language is chosen.

Reply:
Done.